src/Sql/Sql/DataClassification/Services/DataClassificationAdapter.cs

```diff
@@ -285,12 +285,20 @@ private static SensitivityLabel ToSensitivityLabel(SensitivityLabelModel sensiti
 
         private static SensitivityLabelModel ToSensitivityLabelModel(SensitivityLabel sensitivityLabel)
         {
-            string[] idComponents = sensitivityLabel.Id.Split('/');
+            const string schemas = "/schemas/";
+            const string tables = "/tables/";
+            const string columns = "/columns/";
+            const string sensitivityLabels = "/sensitivityLabels/";
+            int indexOfSchema = sensitivityLabel.Id.IndexOf(schemas);
+            int indexOfTables = sensitivityLabel.Id.IndexOf(tables);
+            int indexOfColumns = sensitivityLabel.Id.IndexOf(columns);
+            int indexOfSensitivityLabels = sensitivityLabel.Id.IndexOf(sensitivityLabels);
+
             return new SensitivityLabelModel
             {
-                SchemaName = idComponents[12],
-                TableName = idComponents[14],
-                ColumnName = idComponents[16],
+                SchemaName = sensitivityLabel.Id.Substring(indexOfSchema + schemas.Length, indexOfTables - indexOfSchema - schemas.Length),
```

Comment:
I suggest to use regex to get the value of these variables. You can find [here](https://github.com/Azure/azure-powershell/blob/de80a8336303eea033e219fb9154463001d70591/src/CustomProviders/generated/api/CustomProviders.cs#L909) an example.

Reply:
@wyunchi-ms Done, thanks for the suggestion!